<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>316</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Curling League Database</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Leagues:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="leagues_listwidget">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="delete_league_button">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Delete League</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="edit_league_button">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Edit League</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_league_button">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Add League</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="new_league_lineedit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Enter League Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_league_button">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Export League</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="import_league_button">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Import League</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="import_league_lineedit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Enter League Name</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="load_database_button">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>&amp;Load Database</string>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save_database_button">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>&amp;Save Database</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>leagues_listwidget</tabstop>
  <tabstop>delete_league_button</tabstop>
  <tabstop>edit_league_button</tabstop>
  <tabstop>add_league_button</tabstop>
  <tabstop>new_league_lineedit</tabstop>
  <tabstop>export_league_button</tabstop>
  <tabstop>import_league_button</tabstop>
  <tabstop>import_league_lineedit</tabstop>
  <tabstop>load_database_button</tabstop>
  <tabstop>save_database_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>new_league_lineedit</sender>
   <signal>returnPressed()</signal>
   <receiver>add_league_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>import_league_lineedit</sender>
   <signal>returnPressed()</signal>
   <receiver>import_league_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leagues_listwidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>edit_league_button</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
